packages/docusaurus-theme-classic/src/theme/BlogPostItem/index.tsx

```diff
@@ -90,7 +90,7 @@ function BlogPostItem(props: Props): JSX.Element {
             {formattedDate}
           </time>
 
-          {readingTime && (
+          {typeof readingTime !== 'undefined' && (
```

Comment:
👍

Reply:
I noticed this when I put up a dummy post with no content and it just showed a weird `August 21, 20210`, so fixed it here